Configuration/Routes/List.yaml

```diff
@@ -4,7 +4,7 @@ routeEnhancers:
     extension: AcademicPersons
     plugin: List
     routes:
-      - routePath: '/{page}'
+      - routePath: 'page-{page}'
```

Comment:
Ideally, the text "page" will be translated: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Routing/AdvancedRoutingConfiguration.html#localemodifier